scripts/logfile-parser.sh

```diff
@@ -37,14 +37,40 @@ LogParser() {
             SendDiscordMessage "$DISCORD_PLAYER_LEAVE_TITLE" "$message" "$DISCORD_PLAYER_LEAVE_COLOR"
         fi
 
-        if [[ "$line" == "Started session with Game ID "* ]]; then
+        if [[ "$line" == "Started session with info: "* ]]; then
             [[ "${DISCORD_SERVER_START_ENABLED,,}" == false ]] && continue
 
-            # Extract Game ID
-            gameid=$(echo "$line" | awk '{print $6}')
+            game_info="${STEAMAPPDIR}/GameInfo.txt"
+
+            read -r gameid \
+                    allowed_platforms \
+                    public_ip \
+                    port \
+                    password \
+                    join_string < <(
+              awk -F': ' '
+                BEGIN { gameid=""; allowed_platforms=""; public_ip=""; port=""; password=""; join_string="" }
+                /^GameID:/ { gameid=$2 }
+                /^Allowed platforms:/ { allowed_platforms=$2 }
+                /^Public IP:/ { public_ip=$2 }
+                /^Port:/ { port=$2 }
+                /^Password:/ { password=$2 }
+                /^Paste to ip-field/ { if (getline) join_string=$0 }
```

Comment:
Relying on the next line to extract the join string rather than regex to avoid parsing challenges between IPv4/IPv6

Reply:
I think there's a higher chance the format of `GameInfo.txt` changes in the future than the join string format, so let's go with building the string instead?

Reply:
> building the string

Added in https://github.com/escapingnetwork/core-keeper-dedicated/pull/98/commits/6352cd337599b2960ea7d324f6e715cade88ef9f